wlct/tournaments.py

```diff
@@ -2708,7 +2711,7 @@ def process_new_games(self):
 
         # cache all the games we know about so far that are in progress
         team_game_data = defaultdict(list)
-        games = TournamentGame.objects.filter(tournament=self, is_finished=False)
+        games = TournamentGame.objects.filter(tournament=self)
```

Comment:
Changed this as I wasnt entirely sure why only unfinished games were checked. Should include all games to make sure all pairs are found